### DISCUSSION TITLE
Make the Devicon API project official and list it in the Wiki

### DISCUSSION THREAD

**Discussion by Panquesito7** [devicons/devicon#1672] [Ideas]
@lunatic-fox has made a great tool for having customized icons, such as light mode, dark mode, and even re-sizing the SVGs!
This is very useful, and I definitely think we should make this resource official and list it in our Wiki.

Project link: https://github.com/lunatic-fox/deviconapi
Original discussion: https://github.com/devicons/devicon/issues/1383

**Comment by Snailedlt** [comment]
Agreed!
@lunatic-fox any thoughts? :)

**Comment by lunatic-fox** [comment]
I loved it! 🤩
However, I will, until this weekend, take a look back into this project, just to make sure that everything is ok! 🙂

**Comment by lunatic-fox** [comment]
Made some change here and there! 🛠
I think it's okay now. Thank you!!! 🤩

**Reply by Panquesito7** [reply]
Great, thanks! :)
How do you think we should proceed?

- Just resource it on the Wiki.
- Give Devicon maintainers write access in case a change is needed or such.
- Move the repository to the Devicon organization. You will still be the main maintainer with full access to the repository.
- Any other ideas?

**Reply by lunatic-fox** [reply]
Because it's a small project that doesn't change frequently (most on Devicon version updates), I think, by now, we should start just by resourcing it on the Wiki. And by the way it gets bigger (like in complexity) we can move to Devicon organization. 😁

**Reply by Panquesito7** [reply]
Cool! Feel free to add it yourself to the Wiki. We can make improvements later on. 🙂

**Reply by Snailedlt** [reply]
Btw @lunatic-fox have you taken a look at maltejur's website + API? Maybe you could get some features from it?

https://github.com/maltejur/devicon-website
https://devicon-website.vercel.app/
